### DISCUSSION TITLE
First Version on KE 1.5.X

### DISCUSSION THREAD

**Discussion by DannybotDev** [FunnyDannyG/FNFMegaMayhem#1] [Announcements]
This version was ported to a newer version of kade engine and is also much smaller in file size due to me removing the unused songs from the base games weeks 1-6.

<hr /><em>This discussion was created from the release <a href='https://github.com/DannybotDev/FNFMegaMayhem/releases/tag/v1.0.0-prealpha'>First Version on KE 1.5.X</a>.</em>

**Comment by JordiWifBread** [comment]
slarp in flarpy shit bart fart carty esyrh
